projects/snapshot_bot/config.cc

```diff
@@ -0,0 +1,271 @@
+#include "config.h"
```

Comment:
Splitting that awful header into a cc is an excellent move and must do wonders for the compile time...